src/index.d.ts

```diff
@@ -0,0 +1,65 @@
+import * as resultTypes from "./types";
+
+declare namespace PlantUMLParser {
+  export class Class {
+    constructor(name: string, title: string, isAbstract: boolean, members: object[]);
+  }
+
+  export class Component {
+    constructor(name: string, title: string);
+  }
+
+  export class Enum {
+    constructor(name: string, title: string, members: object[]);
+  }
+
+  export class File {
+    constructor(name: string, diagrams: object[]);
+  }
+
+  export class Group {
+    constructor(name: string, title: string, type: string, elements: object[]);
+  }
+
+  export class Interface {
+    constructor(name: string, title: string, members: object[]);
+  }
+
+  export class MemberVariable {
+    constructor(name: string, isStatic: boolean, accessor: string, type: string);
+  }
+
+  export class Method {
+    constructor(name: string, isStatic: boolean, accessor: string, returnType: string, _arguments?: string);
+  }
+
+  export class Relationship {
+    constructor(left: string, right: string, leftType: string, rightType: string, leftArrowHead: string, rightArrowHead: string, leftArrowBody: string, rightArrowBody: string, leftCardinality: string, rightCardinality: string, label: string);
+  }
+
+  export class UML {
+    constructor(elements: [PlantUMLParser.Class, PlantUMLParser.Class, PlantUMLParser.Relationship]);
```

Comment:
should this not be: `(PlantUMLParser.Class | PlantUMLParser.Relationship )[]` ?
Note: there are more possible types in this array than just `Class` and `Relationship` see [`UMLElement`](https://github.com/Enteee/plantuml-parser/blob/d9c001d3b72e648a52becbd19e17f0015bafcc77/src/plantuml.pegjs#L40).